src/main/java/org/sagebionetworks/bridge/config/Environment.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.config;
+
+public enum Environment {
+    LOCAL,
+    DEV,
+    DEVELOP,
+    DEVELOPMENT,
+    STAGING,
+    UAT,
+    PROD,
+    PRODUCTION
```

Comment:
I was thinking about the problem too.  Another way to break away from it would be not to define the set environments at all.  Users of the library would then have the freedom to define any environment they want.  I am still using the enum to be backward compatible but adding more labels to make it more accommodating.